Dockerfile

```diff
@@ -28,5 +28,10 @@ COPY ./cli ./cli
 COPY ./.git ./.git
 COPY ./.pre-commit-config.yaml ./.flake8 ./.gitignore ./
 
+# Pre-download the model
+ENV PYTHONPATH "${PYTHONPATH}:/app"
+RUN python '/app/cli/warm_up_model.py'
```

Comment:
FYI: The backend does something similar without the need for an extra python file: https://github.com/climatepolicyradar/navigator-backend/blob/b5488c40483cc55a39215e86f308221413359c64/Dockerfile#L36